<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgColorSpaceSelector</class>
 <widget class="QWidget" name="WdgColorSpaceSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>101</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblColorSpaces">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Model:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>cmbColorModels</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KisCmbIDList" name="cmbColorModels">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblColorModels">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Depth:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>cmbColorModels</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KisCmbIDList" name="cmbColorDepth">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblProfiles">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Profile:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>cmbProfile</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SqueezedComboBox" name="cmbProfile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnInstallProfile">
       <property name="toolTip">
        <string>Install a new profile from a file</string>
       </property>
       <property name="whatsThis">
        <string>Install a new profile from a file.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnDownloadProfile">
       <property name="toolTip">
        <string>Install a new profile from a file</string>
       </property>
       <property name="whatsThis">
        <string>Install a new profile from a file.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnUploadProfile">
       <property name="toolTip">
        <string>Install a new profile from a file</string>
       </property>
       <property name="whatsThis">
        <string>Install a new profile from a file.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends>QComboBox</extends>
   <header>widgets/kis_cmb_idlist.h</header>
  </customwidget>
  <customwidget>
   <class>SqueezedComboBox</class>
   <extends>QComboBox</extends>
   <header>squeezedcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
